pkg/core/region.go

```diff
@@ -2142,7 +2142,7 @@ func HexRegionKey(key []byte) []byte {
 // HexRegionKeyStr converts region key to hex format. Used for formatting region in
 // logs.
 func HexRegionKeyStr(key []byte) string {
-	return typeutil.BytesToString(HexRegionKey(key))
+	return string(HexRegionKey(key))
```

Comment:
could you refer to relative go changes?

Reply:
See https://github.com/golang/go/issues/53003#issuecomment-1771118044